stock_inventory_import/wizard/import_inventory.py

```diff
@@ -14,7 +14,7 @@ def _get_default_location(self):
         if 'active_id' in ctx:
             inventory_obj = self.env['stock.inventory']
             inventory = inventory_obj.browse(ctx['active_id'])
-        return inventory.location_id
+            return inventory.location_id
```

Comment:
ctx = self.env.context
if 'active_id' in ctx:
    inventory_obj = self.env['stock.inventory']
    inventory = inventory_obj.browse(ctx['active_id'])
return inventory.location_id or self.env['stock.location']